modules/s3/locals.tf

```
@@ -0,0 +1,10 @@
+locals {
+  # The name of the bucket
+  bucket_name = "devops-ic-bucket"
```

the app can have different buckets for different things (assets, just files, something else)
```suggestion
  bucket_name = "devops-ic-assets-bucket"
```